client/views/login.jade

```diff
@@ -19,4 +19,7 @@ html(lang='en')
                 a(href='/auth/google').btn.btn-block.btn-social.btn-google-plus.sign-in
                     i.fa.fa-google-plus
                     | Sign in with Google
+                a(href='/auth/azureadoauth2').btn.btn-block.btn-social.btn-windows.sign-in

```

Comment:
can we change this so that it shows only the button for the login-scheme that is enabled? You can't enable both Google Auth *and* Azure Auth at the same time, so it doesn't make sense to show both buttons